packages/nx-playwright/src/generators/project/generator.ts

```diff
@@ -30,6 +33,24 @@ export default async function (host: Tree, options: NxPlaywrightGeneratorSchema)
     sourceRoot: `${normalizedOptions.projectRoot}/src`,
     projectType: 'application',
     targets: {
+      axe: options.includeAxe
+        ? {
+            executor: '@mands/nx-playwright:playwright-executor',
+            options: {
+              e2eFolder: normalizedOptions.projectRoot,
+              devServerTarget: options.project ? `${options.project}:serve` : undefined,
+              path: './axe-tests',
```

Comment:
Is the only difference between axe config and e2e config the path? If so it may not be worth creating a new target?

Reply:
Yes. It's the only difference. I've updated the PR so now `axe` is a configuration for the `e2e` target.